init/main.c

```diff
@@ -88,1002 +86,9 @@
lename),(constchar__user*const__user*)argv_init,(constchar__user*const__user*)envp_init);}staticinttry_to_run_init_process(constchar*init_filename){intret;ret=run_init_process(init_filename);if(ret&&ret!=-ENOENT){pr_err("Startinginit:%sexistsbutcouldn'texecuteit(error%d)\n",init_filename,ret);}returnret;}staticnoinlinevoid__initkernel_init_freeable(void);#ifdefined(CONFIG_STRICT_KERNEL_RWX)||defined(CONFIG_STRICT_MODULE_RWX)boolrodata_enabled__ro_after_init=true;staticint__initset_debug_rodata(char*str){returnstrtobool(str,&rodata_enabled);}__setup("rodata=",set_debug_rodata);#endif#ifdefCONFIG_STRICT_KERNEL_RWXstaticvoidmark_readonly(void){if(rodata_enabled){mark_rodata_ro();rodata_test();}elsepr_info("Kernelmemoryprotectiondisabled.\n");}#elsestaticinlinevoidmark_readonly(void){pr_warn("Thisarchitecturedoesnothavekernelmemoryprotection.\n");}#endifstaticint__refkernel_init(void*unused){intret;kernel_init_freeable();/*needtofinishallasync__initcodebeforefreeingthememory*/async_synchronize_full();ftrace_free_init_mem();free_initmem();mark_readonly();system_state=SYSTEM_RUNNING;numa_default_policy();rcu_end_inkernel_boot();if(ramdisk_execute_command){ret=run_init_process(ramdisk_execute_command);if(!ret)return0;pr_err("Failedtoexecute%s(error%d)\n",ramdisk_execute_command,ret);}/**Wetryeachoftheseuntilonesucceeds.**TheBourneshellcanbeusedinsteadofinitifweare*tryingtorecoverareallybrokenmachine.*/if(execute_command){ret=run_init_process(execute_command);if(!ret)return0;panic("Requestedinit%sfailed(error%d).",execute_command,ret);}if(!try_to_run_init_process("/sbin/init")||!try_to_run_init_process("/etc/init")||!try_to_run_init_process("/bin/init")||!try_to_run_init_process("/bin/sh"))return0;panic("Noworkinginitfound.Trypassinginit=optiontokernel.""SeeLinuxDocumentation/admin-guide/init.rstforguidance.");}staticnoinlinevoid__initkernel_init_freeable(void){/**Waituntilkthreaddisallset-up.*/wait_for_completion(&kthreadd_done);/*Nowtheschedulerisfullysetupandcandoblockingallocations*/gfp_allowed_mask=__GFP_BITS_MASK;/**initcanallocatepagesonanynode*/set_mems_allowed(node_states[N_MEMORY]);cad_pid=task_pid(current);smp_prepare_cpus(setup_max_cpus);workqueue_init();init_mm_internals();do_pre_smp_initcalls();lockup_detector_init();smp_init();sched_init_smp();page_alloc_init_late();do_basic_setup();/*Openthe/dev/consoleontherootfs,thisshouldneverfail*/if(sys_open((constchar__user*)"/dev/console",O_RDWR,0)<0)pr_err("Warning:unabletoopenaninitialconsole.\n");(void)sys_dup(0);(void)sys_dup(0);/**checkifthereisanearlyuserspaceinit.Ifyes,letitdoall*thework*/if(!ramdisk_execute_command)ramdisk_execute_command="/init";if(sys_access((constchar__user*)ramdisk_execute_command,0)!=0){ramdisk_execute_command=NULL;prepare_namespace();}/**Ok,wehavecompletedtheinitialbootup,and*we'reessentiallyupandrunning.Getridofthe*initmemsegmentsandstarttheuser-modestuff..**rootfsisavailablenow,tryloadingthepublickeys*anddefaultmodules*/integrity_load_keys();load_default_modules();}
```

Comment:
Don't forget the comments 👍

Reply:
I mean come on guys the submitter is definitely retard. Who could read the code look like a js compiled. He should read at least basics of the K&R book or read anything about C and C++ comments and empty lines as everywhere are ignored. There is no point spamming my inbox although the tread was a fun joke on a pull request submitter its annoying now. I woke up and go over 60 messages from github torvalds 437 I mean wtf guys. Stop it. :)

Reply:
HE FORGOT TO REMOVE THE COMMENTS!